config/HH_bbtautau/Run3_2022/samples.yaml

```
@@ -0,0 +1,350 @@
+  sampleType: GluGluToRadion
+  mass: 250
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-260:
+  sampleType: GluGluToRadion
+  mass: 260
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-270:
+  sampleType: GluGluToRadion
+  mass: 270
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-280:
+  sampleType: GluGluToRadion
+  mass: 280
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-300:
+  sampleType: GluGluToRadion
+  mass: 300
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-350:
+  sampleType: GluGluToRadion
+  mass: 350
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-450:
+  sampleType: GluGluToRadion
+  mass: 450
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-550:
+  sampleType: GluGluToRadion
+  mass: 550
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-600:
+  sampleType: GluGluToRadion
+  mass: 600
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-650:
+  sampleType: GluGluToRadion
+  mass: 650
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-700:
+  sampleType: GluGluToRadion
+  mass: 700
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-800:
+  sampleType: GluGluToRadion
+  mass: 800
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-1000:
+  sampleType: GluGluToRadion
+  mass: 1000
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-1200:
+  sampleType: GluGluToRadion
+  mass: 1200
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-1400:
+  sampleType: GluGluToRadion
+  mass: 1400
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-1600:
+  sampleType: GluGluToRadion
+  mass: 1600
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-1800:
+  sampleType: GluGluToRadion
+  mass: 1800
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-2000:
+  sampleType: GluGluToRadion
+  mass: 2000
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-2500:
+  sampleType: GluGluToRadion
+  mass: 2500
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-3000:
+  sampleType: GluGluToRadion
+  mass: 3000
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-4000:
+  sampleType: GluGluToRadion
+  mass: 4000
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Tau_M-5000:
+  sampleType: GluGluToRadion
+  mass: 5000
+  spin: 0
+  generator: madgraph
+  crossSection: 1pb
+GluGlutoBulkGravitontoHHto2B2Tau_M-250:
+  sampleType: GluGlutoBulkGraviton
```

```suggestion
  sampleType: GluGluToBulkGraviton
```
same in all other places